MAUI/MauiAppBasic/MSALClient/PCAWrapper.cs

```diff
@@ -0,0 +1,108 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Identity.Client;
+
+namespace MauiAppBasic.MSALClient
+{
+    /// <summary>
+    /// This is a wrapper for PCA. It is singleton and can be utilized by both application and the MAM callback
+    /// </summary>
+    public class PCAWrapper
+    {
+        /// <summary>
+        /// This is the singleton used by consumers
+        /// </summary>
+        public static PCAWrapper Instance { get; private set; } = new PCAWrapper();

```

Comment:
Consider using `Lazy` for singletoin initialization https://csharpindepth.com/articles/singleton#cctor

Reply:
Thanks for sharing the article and the thought. Since PCAWrapper constructor does not have perf or memory issue, Lazy is not required. (Updated comment). Lazy will add unnecessary code.